apache/superset
# feat(can_share): can share chart and dashboard

### SUMMARY
As all things are so should sharing be a managed under a permission the user entitles to.
can_share_dashboard and can_share_chart are the new permissions .
both hide/show `ShareMenuItems` component on the dashboard/chart menus

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/47772523/114314495-a166ad00-9b03-11eb-9b10-604056ee85e8.mov



### TEST PLAN
1. check that sharing is menu option is shown by default for Admin/Gamma/Alpha
2. remove can share chart/dashboard on Superset from you user role
3. check that sharing menu option is hidden 

### ADDITIONAL INFORMATION
